app/views/admin/AdministerUserModal.coffee

```diff
@@ -265,6 +265,19 @@ module.exports = class AdministerUserModal extends ModalView
 
   onClickSchoolAdminCheckbox: (e) ->
     checked = @$(e.target).prop('checked')
+    cancelled = false
+    if checked
+      unless window.confirm("ENABLE school administator for #{@user.get('email') || @user.broadName()}?")
```

Comment:
Perhaps I am misunderstanding the wording here, but would it not be `DISABLE` here when the checkbox is in a checked state?

Reply:
`checked` seems to be the new state, which gets discarded by `preventDefault`.  Kinda weird.